### ISSUE TITLE
Shrinker missing positional argument

### ISSUE THREAD

**Issue by tmr232** [Zac-HD/hypofuzz#21]
I tried running hypofuzz, and this is the error I'm getting:

```
Traceback (most recent call last):
  File "C:\Users\tamir\AppData\Local\Programs\Python\Python311\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\tamir\AppData\Local\Programs\Python\Python311\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Code\crafting-interpreters-py\.venv\Lib\site-packages\hypofuzz\interface.py", line 91, in _fuzz_several
    fuzz_several(*tests)
  File "C:\Code\crafting-interpreters-py\.venv\Lib\site-packages\hypofuzz\hy.py", line 378, in fuzz_several
    t.run_one()
  File "C:\Code\crafting-interpreters-py\.venv\Lib\site-packages\hypofuzz\hy.py", line 203, in run_one
    shrinker = Shrinker(
               ^^^^^^^^^
TypeError: Shrinker.__init__() missing 1 required positional argument: 'explain'
```

I'm running:
- Windows 11
- Python 3.11
- hypofuzz 23.4.1

**Comment by Zac-HD** [comment]
Gah, yeah, recent changes to Hypothesis have broken this - aiming to get a fix out this weekend.

**Comment by Zac-HD** [comment]
Fixed in HypoFuzz version 23.05.1 - thanks again for the report ❤️
